src/GCPClientManager.php

```diff
@@ -122,6 +136,7 @@ public function getBigQueryClient(string $runId, GenericBackendCredentials $cred
         return new BigQueryClientWrapper($runId, [
             'keyFile' => CredentialsHelper::getCredentialsArray($credentials),
             'restRetryFunction' => Retry::getRetryDecider($this->logger),
+            'requestTimeout' => 60,
```

Comment:
Tady nemožeš dat takový timeout utne ti to po 60s třeba import do tabulky, asi. Tam bude potřeba něco jako 12h nebo tak něco.

Reply:
ja som to pochopil, ze je to akoze fakt len request timeout a ze query moze bezat jak dlho chce
```
 @type float $requestTimeout Seconds to wait before timing out the
     *           request. **Defaults to** `0` with REST and `60` with gRPC.
```

Reply:
ale nejake napad ako to celkovo otestovat @zajca ? Pozeral som xdebugo a dost requestov to maju vyplnene a ked mam zmeny z toho PR tak sa na tych miestach pridaju tie nove hodnoty